plugins/github-actions/dev/index.tsx

```diff
@@ -14,7 +14,65 @@
  * limitations under the License.
  */
 
+import React from 'react';
 import { createDevApp } from '@backstage/dev-utils';
-import { githubActionsPlugin } from '../src/plugin';
+import { EntityGithubActionsContent, githubActionsPlugin } from '../src/plugin';
+import { EntityProvider } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+import { GithubActionsApi, githubActionsApiRef } from '../src';
+import getWorkflowRunResponse from './getWorkflowRun.json';
+import listJobsForWorkflowRunResponse from './listJobsForWorkflowRun.json';
 
-createDevApp().registerPlugin(githubActionsPlugin).render();
+const mockEntity: Entity = {
+  apiVersion: 'backstage.io/v1alpha1',
+  kind: 'Component',
+  metadata: {
+    name: 'backstage',
+    description: 'backstage.io',
+    annotations: {
+      'github.com/project-slug': 'backstage/backstage',
+    },
+  },
+  spec: {
+    lifecycle: 'production',
+    type: 'website',
+    owner: 'user:guest',
+  },
+};
+
+const mockGithubActionsApi: GithubActionsApi = {
```

Comment:
I only implemented the methods that were needed to present the UI elements I updated. I tried implementing all of them at first, but it seems like the types from `@octokit/rest` are out of sync with the example responses on https://docs.github.com/en/rest/reference/actions